HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -873,10 +873,18 @@ private static string GetLocalAuthorityNotParticipatingMessagePartialViewPath(Qu
     
     private static string GetLocalAuthorityConfirmationMessagePartialViewPath(Questionnaire questionnaire)
     {
-        var partialViewName = questionnaire.CustodianCode switch
+        var partialViewName = "";
+        if (questionnaire.LocalAuthorityHug2Status == LocalAuthorityData.Hug2Status.Pending)
         {
-            _ => "Default"
-        };
+            partialViewName = "Pending";
+        }
+        else
+        {
+             partialViewName = questionnaire.CustodianCode switch
+            {
+                _ => "Default"

```

Comment:
Similar question to the other PR about whether we expect other cases here - it would tidy things up quite a bit here!

Reply:
For this one, I wasn't told to account for LA-specific messages, just to add a version which would be shown to all Pending LA's.
This switch was here prior to my changes, so I preserved the functionality for non-pending LA's to have customised messages.

Reply:
You can combine the `switch` statement here to pattern match on both the status and the code:

```
var partialViewName = (questionnaire.LocalAuthorityHug2Status, questionnaire.CustodianCode) switch
{
    (LocalAuthorityData.Hug2Status.Pending, _) => "Pending",
    _ => "Default"
};
```